### ISSUE TITLE
5025 (or earlier): Savegame compatibility broken

### ISSUE THREAD

**Issue by panther2** [triplea-game/triplea#1943]
Noted with 1.9.0.0.5025, but may have started earlier.

Steps to reproduce: 
Load any savegame from forum games (recently) started. 
Hit "Start Local Game" or "Start PBEM Game".

Console:

```
triplea.engine.version.bin:1.9
java.lang.NullPointerException
	at games.strategy.engine.framework.startup.ui.PlayerSelectorRow.setWidgetActivation(PlayerSelectorRow.java:159)
	at games.strategy.engine.framework.startup.ui.PlayerSelectorRow.<init>(PlayerSelectorRow.java:103)
	at games.strategy.engine.framework.startup.ui.SetupPanel.layoutPlayerComponents(SetupPanel.java:131)
	at games.strategy.engine.framework.startup.ui.LocalSetupPanel.<init>(LocalSetupPanel.java:29)
	at games.strategy.engine.framework.startup.mc.SetupPanelModel.showLocal(SetupPanelModel.java:37)
	at games.strategy.engine.framework.startup.ui.MetaSetupPanel.lambda$setupListeners$204(MetaSetupPanel.java:118)
	at javax.swing.AbstractButton.fireActionPerformed(Unknown Source)
	at javax.swing.AbstractButton$Handler.actionPerformed(Unknown Source)
	at javax.swing.DefaultButtonModel.fireActionPerformed(Unknown Source)
	at javax.swing.DefaultButtonModel.setPressed(Unknown Source)
	at javax.swing.plaf.basic.BasicButtonListener.mouseReleased(Unknown Source)
	at org.pushingpixels.substance.internal.utils.RolloverButtonListener.mouseReleased(RolloverButtonListener.java:124)
	at java.awt.AWTEventMulticaster.mouseReleased(Unknown Source)
	at java.awt.Component.processMouseEvent(Unknown Source)
	at javax.swing.JComponent.processMouseEvent(Unknown Source)
	at java.awt.Component.processEvent(Unknown Source)
	at java.awt.Container.processEvent(Unknown Source)
	at java.awt.Component.dispatchEventImpl(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.processMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.dispatchEvent(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Window.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$500(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at games.strategy.engine.framework.GameRunner$1.dispatchEvent(GameRunner.java:310)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
java.lang.NullPointerException
	at games.strategy.engine.framework.startup.ui.PlayerSelectorRow.setWidgetActivation(PlayerSelectorRow.java:159)
	at games.strategy.engine.framework.startup.ui.PlayerSelectorRow.<init>(PlayerSelectorRow.java:103)
	at games.strategy.engine.framework.startup.ui.SetupPanel.layoutPlayerComponents(SetupPanel.java:131)
	at games.strategy.engine.framework.startup.ui.LocalSetupPanel.<init>(LocalSetupPanel.java:29)
	at games.strategy.engine.framework.startup.mc.SetupPanelModel.showLocal(SetupPanelModel.java:37)
	at games.strategy.engine.framework.startup.ui.MetaSetupPanel.lambda$setupListeners$204(MetaSetupPanel.java:118)
	at javax.swing.AbstractButton.fireActionPerformed(Unknown Source)
	at javax.swing.AbstractButton$Handler.actionPerformed(Unknown Source)
	at javax.swing.DefaultButtonModel.fireActionPerformed(Unknown Source)
	at javax.swing.DefaultButtonModel.setPressed(Unknown Source)
	at javax.swing.plaf.basic.BasicButtonListener.mouseReleased(Unknown Source)
	at org.pushingpixels.substance.internal.utils.RolloverButtonListener.mouseReleased(RolloverButtonListener.java:124)
	at java.awt.AWTEventMulticaster.mouseReleased(Unknown Source)
	at java.awt.Component.processMouseEvent(Unknown Source)
	at javax.swing.JComponent.processMouseEvent(Unknown Source)
	at java.awt.Component.processEvent(Unknown Source)
	at java.awt.Container.processEvent(Unknown Source)
	at java.awt.Component.dispatchEventImpl(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.processMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.dispatchEvent(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Window.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$500(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at games.strategy.engine.framework.GameRunner$1.dispatchEvent(GameRunner.java:310)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
```

**Comment by RoiEXLab** [comment]
Noticed this as well, but I was on a development branch with a lot of things changed, so I thought this was the problem...

**Comment by RoiEXLab** [comment]
@ron-murhammer I'm guessing this was introduced by https://github.com/triplea-game/triplea/commit/8ae62071cfd01415ace79187544e07b3b01699e7#diff-de5d14e5f1cfa9590384a5d09c8e1cb2

**Comment by ron-murhammer** [comment]
@panther2 I tested a save game I had which reproduced the error and it should now be fixed with the latest pre-release that has #1946

**Comment by panther2** [comment]
Confirmed, using 5043, thanks.
